accelerator/real_accelerator.py

```diff
@@ -44,17 +44,41 @@ def get_accelerator():
         else:
             ds_accelerator = XPU_Accelerator()
             _validate_accelerator(ds_accelerator)
+            print(f"Setting ds_accelerator to {ds_accelerator._name} (auto-detect)")
             return ds_accelerator
 
-        from .cuda_accelerator import CUDA_Accelerator
-        ds_accelerator = CUDA_Accelerator()
-        _validate_accelerator(ds_accelerator)
-    return ds_accelerator
+        # We need a way to choose between CUDA_Accelerator and CPU_Accelerator
+        # Currently we detect whether intel_etension_for_pytorch is installed
+        # in the environment and use CPU_Accelerator if the answewr is True.
+        # An alternative might be detect whether CUDA device is installed on
+        # the system but this comes with two pitfalls:
+        # 1. the system may not have torch pre-installed, so
+        #    get_accelerator().is_avaiable() may not work.
+        # 2. Some scenario like install on login node (without CUDA device)
+        #    and run on compute node (with CUDA device) may cause mismatch
+        #    between installation time and runtime.
+        try:
```

Comment:
Also need to add mechanism to override auto detection.  The logic should be controlled by environment variable:
DS_USE_CUDA_ACCELERATOR ==> CUDA path
DS_USE_CPU_ACCELERATOR ==> CPU path
DS_USE_XPU_ACCELERATOR ==> XPU path
This could allow user to have a more explicit way to use certain accelerator

Reply:
Done.